src/include/str_list.h

```diff
@@ -7,22 +7,103 @@
 #include <string>
 #include <vector>
 
+
+/**
+ * Parse **str* and set **str_list** using these delimiters : ";,= \t"
```

Comment:
It would rather be something like 

   Split **str** into a list of strings, using the **";,= \t"** delimiters and output the result in **str_list**.

It is correct to say the **str** is parsed but you should be more precise about what is done with the parsing.